manifests/params.pp

```diff
@@ -487,6 +491,7 @@
     $fastcgi_lib_path       = '/var/lib/apache2/fastcgi'
     $mime_support_package = 'aaa_base'
     $mime_types_config    = '/etc/mime.types'
+    $mime_types_additional = {'AddHandler' => {'type-map' => 'var'}, 'AddType' => {'type-map' => '.shtml'},'AddOutputFilter' => {'INCLUDES' => '.shtml'},}
```

Comment:
why are you putting this in every single branch, instead of just on top?
It's always the same, right?